test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2653,34 +2661,58 @@ UCS_TEST_P(test_ucp_sockaddr_protocols, tag_rndv_unexp, "RNDV_THRESH=10k")
 
 UCS_TEST_P(test_ucp_sockaddr_protocols, stream_bcopy_4k_exp, "ZCOPY_THRESH=inf")
 {
+    if (m_ucp_config->ctx.proto_enable) {
+        UCS_TEST_SKIP_R("ProtoV2 is not implemented");
+    }
+
     test_stream_send_recv(4 * UCS_KBYTE, true);
 }
 
 UCS_TEST_P(test_ucp_sockaddr_protocols, stream_bcopy_4k_unexp,
            "ZCOPY_THRESH=inf")
 {
+    if (m_ucp_config->ctx.proto_enable) {
```

Comment:
Maybe we can use `UCS_TEST_SKIP_COND_P` instead?

Reply:
Inside `test_stream_send_recv` seems like the right place.